api/src/api/members.js

```diff
@@ -72,6 +72,74 @@ router.post(
   }),
 );
 
+// Gives the enum options to the frontend to populate the dropdown.
+// The key in the 'options' object must be the same as the DB attribute.
+router.get(
+  '/options',
+  requireRegistered,
+  errorWrap(async (req, res) => {
+    const options = {
+      level: Member.levelEnum,
+      location: Member.locationEnum,
+      role: Member.roleEnum,
+      gradSemester: Member.semesterEnum,
+      generationSemester: Member.semesterEnum,
+      classStanding: Member.classStandingEnum,
+      status: Member.statusEnum,
+    };
+
+    // Label all items with 'label' and 'value'
+    const labeledOptions = {};
+    for (var attributeLabel in options) {
+      if (!Object.prototype.hasOwnProperty.call(options, attributeLabel))
+        continue;
+
+      labeledOptions[attributeLabel] = [];
```

Comment:
This could be `labeledOptions[attributeLabel] = optionsEnum.map((option) => { label: option, value: option });` and can replace lines 98-108

Reply:
Just wondering, did you include label so we can specify the pretty-printed name from the backend?

We can figure this out later, but I know @amit-sawhney already added those header names in `Table.js`, so we should decide where we store those names so that both the table and the member profile page use one source of info for them.

Reply:
Another approach you might want to consider is only giving the `value`, and leaving `label` up to the frontend, similar to how we leave the "label" of `attributeLabel` up to the frontend, too.

Reply:
The only issue with letting the frontend decide the label is that the frontend would then need to hardcode a field for each enum value. That would defeat the point of getting them from the backend.

Reply:
Yes, great point. This is also a good argument for letting the backend decide the labeled names for each of the fields as well, in the table. Although there is some more table-specific configuration there, maybe @amit-sawhney and @alicesf2 can weigh in.